GitFyle.Core.Api/Models/Foundations/Contributions/Contribution.cs

```diff
@@ -17,6 +17,10 @@ public class Contribution
         public Guid ContributionTypeId { get; set; }
         public string ExternalId { get; set; }
         public string Title { get; set; }
+        public string CreatedBy { get; set; }
+        public DateTimeOffset CreatedWhen { get; set; }
```

Comment:
Why do we add the CreatedWhen field? It is not part of the Data model originally setup.